index.js

```diff
@@ -1,13 +1,42 @@
 'use strict'
 
+const back = require('./lib/back')
+const emitter = require('./lib/global_emitter')
+const {
+  activate,
+  isActive,
+  isDone,
+  pendingMocks,
+  activeMocks,
+  removeInterceptor,
+  disableNetConnect,
+  enableNetConnect,
+  removeAll: cleanAll,
+} = require('./lib/intercept')
 const recorder = require('./lib/recorder')
-module.exports = require('./lib/scope')
+const { Scope, load, loadDefs, define } = require('./lib/scope')
 
-module.exports.recorder = {
-  rec: recorder.record,
-  clear: recorder.clear,
-  play: recorder.outputs,
-}
+module.exports = (...args) => new Scope(...args)
```

Comment:
I know it's redundant, but can we fill these in? `basePath, options`
It will help editors with autocomplete and folks digging around who aren't familiar with the code.